<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow.title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="chatTab">
       <attribute name="title">
        <string>tab.chat</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>chat.channels</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listViewChannels">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QStackedWidget" name="stackedWidgetChat">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>chat.onlineUsers</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listViewUsers">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filesTab">
       <attribute name="title">
        <string>tab.files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>menu.file</string>
    </property>
    <addaction name="actionLogOut"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMenu_conversation">
    <property name="title">
     <string>menu.conversation</string>
    </property>
    <addaction name="actionJoinChannel"/>
    <addaction name="actionPartChannel"/>
    <addaction name="actionRefreshChannels"/>
    <addaction name="actionShowPrivateMessages"/>
   </widget>
   <widget class="QMenu" name="administrationMenu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>menu.administration</string>
    </property>
    <addaction name="actionSendQuery"/>
    <addaction name="actionRegisterUser"/>
    <addaction name="actionChangeUserMode"/>
    <addaction name="actionChangeChannelMode"/>
    <addaction name="actionChangePassword"/>
   </widget>
   <widget class="QMenu" name="languageMenu">
    <property name="title">
     <string>menu.language</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMenu_conversation"/>
   <addaction name="administrationMenu"/>
   <addaction name="languageMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogOut"/>
   <addaction name="actionRefreshChannels"/>
   <addaction name="actionShowPrivateMessages"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>action.exit</string>
   </property>
  </action>
  <action name="actionJoinChannel">
   <property name="text">
    <string>action.joinChannel</string>
   </property>
  </action>
  <action name="actionPartChannel">
   <property name="text">
    <string>action.partChannel</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="text">
    <string>action.logOut</string>
   </property>
  </action>
  <action name="actionRefreshChannels">
   <property name="text">
    <string>action.refreshChannels</string>
   </property>
  </action>
  <action name="actionSendQuery">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>action.sendQuery</string>
   </property>
  </action>
  <action name="actionShowPrivateMessages">
   <property name="text">
    <string>action.showPrivateMessages</string>
   </property>
  </action>
  <action name="actionRegisterUser">
   <property name="text">
    <string>action.registerUser</string>
   </property>
  </action>
  <action name="actionChangeUserMode">
   <property name="text">
    <string>action.changeUserMode</string>
   </property>
  </action>
  <action name="actionChangeChannelMode">
   <property name="text">
    <string>action.changeChannelMode</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>action.changePassword</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
